deno.json

```diff
@@ -0,0 +1,40 @@
+{
+    "nodeModulesDir": "manual",
+    "unstable": [
+        "sloppy-imports"
```

Comment:
Alternatively, #77 would obviate this.
This only **effect**s this internally, the built version is fine.

Except for the stack overflow.